lib/external/sentry.js

```diff
@@ -1,3 +1,12 @@
+// Copyright 2018 ODK Central Developers
```

Comment:
It looks like this file was created in 2018: 3d1064060455bc60a193ad003bcde83d7d7ff392. (The file was moved in eca4b52e8eebe52f949ae0fda632d226402f439c.)